### ISSUE TITLE
Data driven discovery of PDEs

### ISSUE THREAD

**Issue by finmod** [JuliaDiffEq/DataDrivenDiffEq.jl#25]
https://advances.sciencemag.org/content/advances/suppl/2017/04/24/3.4.e1602614.DC1/1602614_SM.pdf 

All the elementary functions being in place, this is a natural extension of DataDrivenDiffEq

**Comment by AlCap23** [comment]
Definitely, but as stated in #24, my main focus is getting ISInDy and other methods for ODE and DAE to run. 

But once this is done, I can tackle this.

**Comment by finmod** [comment]
No worry. The sequencing of your workflow may benefit from this approach and steps classification: http://www.dam.brown.edu/people/mmcguirl/DEFD.pdf in developing DataDrivenDiffEq.

The SINDy approach with 4 steps and PDE variant.

The HAVOK approach with 6 steps where Step5 is the Step 4 of SINDy.

**Comment by finmod** [comment]
@AlCap23 These SINDy and iSINDy are major milestones in learning nonlinear models from datasets. Just a gentle reminder for the extension of the examples to the Lorenz PDE.  This would be the cherry on the cake and a fabulous addition to DataDrivenDiffEq.
